apps/gdalalg_vector_sql.h

```diff
@@ -0,0 +1,63 @@
+/******************************************************************************
+ *
+ * Project:  GDAL
+ * Purpose:  "sql" step of "vector pipeline"
+ * Author:   Even Rouault <even dot rouault at spatialys.com>
+ *
+ ******************************************************************************
+ * Copyright (c) 2025, Even Rouault <even dot rouault at spatialys.com>
+ *
+ * SPDX-License-Identifier: MIT
+ ****************************************************************************/
+
+#ifndef GDALALG_VECTOR_SQL_INCLUDED
+#define GDALALG_VECTOR_SQL_INCLUDED
+
+#include "gdalalg_vector_pipeline.h"
+
+//! @cond Doxygen_Suppress
+
+/************************************************************************/
+/*                      GDALVectorSQLAlgorithm                          */
+/************************************************************************/
+
+class GDALVectorSQLAlgorithm /* non final */
+    : public GDALVectorPipelineStepAlgorithm
+{
+  public:
+    static constexpr const char *NAME = "sql";
+    static constexpr const char *DESCRIPTION =
```

Comment:
isn't this potentially localized ?

Reply:
not sure we could do that with a constexpr. Anyway our localization story is inexistent and would be a separated topic